package.json

```diff
@@ -122,7 +122,7 @@
   },
   "prettier": "@prisma-labs/prettier-config",
   "peerDependencies": {
-    "@prisma/client": "^3.5",
+    "@prisma/client": "^3.10",
```

Comment:
Per the other PR, I believe this is a breaking change https://github.com/prisma/nexus-prisma/pull/204/files#r855409916

Reply:
There is no different on the dmmf since 3.10 I added a test for current versions 3.12 and 3.10

Reply:
> Per the other PR, I believe this is a [breaking change](https://github.com/prisma/nexus-prisma/pull/204/files#r855409916) that we need a plan for communicating if we're going to make it. 

I don't know why it's breaking change